lib/services/profiles-api-service.js

```diff
@@ -75,32 +75,29 @@ function profileApiServiceFactory(
         return _.flattenDeep([macs]);
     };
 
-    ProfileApiService.prototype.setLookup = function(req, res) {
-        var query = req.swagger ? req.swagger.query : req.query;
+    ProfileApiService.prototype.getRequestMacAddress = function(query, ipAddress) {
+        if (query.macs && query.ips) {
```

Comment:
it looks like you're changing guard sentences, it reduce 'return' numbers but increase the nesting level, maybe there are debates for this

Reply:
You are right, it's only my preference here, since we only have one nesting level here.